src/core/resources/WindowBorders.qml

```diff
@@ -0,0 +1,38 @@
+import QtQuick 1.1
+import DisplayCluster 1.0
+
+Rectangle {
+    anchors.fill: parent
+    color: "transparent"
+
+    property real baseOpacity: 0.4
+
+    visible: contentwindow.label !== "Dock"
+             && contentwindow.state !== ContentWindow.SELECTED
+    opacity: contentwindow.controlsOpacity * baseOpacity
+
```

Comment:
This component is almost a duplicate of TouchWindowBorders.qml, except for the type of the BorderRectangle objects. I'm wondering if we could use a delegate component property like Qt does to customise the ListView items...